### ISSUE TITLE
Fix fusing patterns for conv that has bias

### ISSUE THREAD

**Issue by mvasiljevicTT** [tenstorrent/tt-mlir#5386]
Unet model has batch_norm ops that come after convolutions that already have bias. After batch norm is decomposed as mul and add op, mul is incorrectly fused and add is not fused at all, as:

```
conv<weight=w, bias=null>
|                            conv<weight=w, bias=b>
add(b)                       |                              conv<weight=a_bn*w, bias=b>
|                      ->    mul(a_bn)                ->    |
mul(a_bn)                    |                              add(b_bn)
|                            add(b_bn)
add(b_bn)
```

and it should be fused as:
```
conv<weight=w, bias=null>
|                            
add(b)                              
|                  ->     . . .    ->   conv<weight=a_bn*w, bias=a_bn*b + b_bn>
mul(a_bn)                           
|                            
add(b_bn)
```
